src/main/java/com/dremio/exec/store/hbase/HBaseScanPrel.java

```diff
@@ -64,7 +64,7 @@ public HBaseScanPrel(
     this.stopRow = stopRow;
   }
 
-  public byte[] getFilter() {
+  public byte[] getHBaseFilter() {
```

Comment:
Let's make this package-private.

Reply:
Made the correct changes and re-ran mvn test, build succeeds and no errors